### DISCUSSION TITLE
Need some help on referencing related tenders

### DISCUSSION THREAD

**Discussion by poetsmeniet** [OP-TED/ted-xml-data-converter#29] [General]
Hi,

I'm trying to relate (for instance) a contract notice with a contract award notice.

According to my discussion on relating documents I could use CBC:CONTRACTFOLDERID field: https://github.com/OP-TED/eForms-SDK/discussions/820

Maybe I misunderstood something there.

I implemented this, and for the eForms it seems to be working well.

But, for converted documents (TED EXports) I get a lot of results that seem unrelated. 

For instance document 727381-2023 has 97 related documents with CONTRACTFOLDERID: aff2863e-b4cc-4e91-baba-b3b85f709117 (in December 2023 alone)

Is this an issue with this field in the converter, or am I making the wrong assumptions?

**Comment by pdonohoe** [comment]
Hi @poetsmeniet , this is because cbc:ContractFolderID (BT-04-notice "Procedure Identifier") is only used in eForms, to link notices using the same procedure together. It is mandatory in eForms, but has no equivalent in TED XML. The converter must put something in this Field when converting a TED notice to eForms. So the fixed value of "aff2863e-b4cc-4e91-baba-b3b85f709117" has been hard-coded. This must be edited before an eForms notice based on this conversion is sent for publication.
To link an eForms notice to a previously published TED notice, please use OPP-090-Procedure  "Previous Notice Identifier". See https://docs.ted.europa.eu/eforms/latest/schema/all-in-one.html#previousNoticeSection
